### PR TITLE
Delete comments

### PR DESCRIPTION
**Pull request by hadleyshapland**
Added functionality to delete comments.
image

### CODE REVIEW

File: portfolio/src/main/webapp/js/scripts.js [hunk 1]
```diff
@@ -61,9 +61,24 @@ function createCommentElement(comment) {
...
+    });
 
     commentElement.appendChild(textElement);
+    commentElement.appendChild(deleteButtonElement);
 
     return commentElement;
+}
+
+/** Tells the server to delete the comment. */
+function deleteComment(comment) {
```
Comment (jerrychangusa) [inline portfolio/src/main/webapp/js/scripts.js]: Consider passing in just the id here if you don't need any other properties.

**Review by jerrychangusa — APPROVED** [review]
LGTM - thanks for keeping the pull request size small!

File: portfolio/src/main/webapp/js/scripts.js (hunk 1 above)
Comment (hadleyshapland) [reply portfolio/src/main/webapp/js/scripts.js]: done!

File: portfolio/src/main/webapp/js/scripts.js [hunk 2]
```diff
@@ -61,9 +61,24 @@ function createCommentElement(comment) {
     commentElement.className = 'comment';
 
     const textElement = document.createElement('span');
-    textElement.innerText = comment.text + " - " + comment.name;
+    textElement.innerText = comment.text + " - " + comment.name + "    ";
+
+    const deleteButtonElement = document.createElement('button');
+    deleteButtonElement.innerText = 'X';
+    deleteButtonElement.addEventListener('click', () => {
```
Comment (ghandre) [inline portfolio/src/main/webapp/js/scripts.js:68]: Ah, this was the other method of adding onclick events we chatted about! (just a remark, no action required)

**Pull request merged**